### PR TITLE
Update bulid_for_windows.sh to use requirements.txt

### PR DESCRIPTION
**Pull request by skyline75489**

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: .circleci/scripts/build_for_windows.sh [hunk 1]
```diff
@@ -31,10 +31,10 @@ eval "$(conda shell.bash hook)"
 conda create -qyn testenv python=3.7
 conda activate testenv
 
-conda install sphinx
-pip install sphinx_gallery==0.3.1 sphinx-copybutton flask pandas spacy ipython scipy pySoundFile scikit-image gym gym-super-mario-bros==7.3.0
-pip install -e git+git://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
-pip install ray[tune] tabulate
+REQUIREMENTS="$(grep -v '^ *#\|^torch\|^torchaudio\|^torchvision|^torchtext' $PROJECT_DIR/requirements.txt  | grep .)"
+echo $REQUIREMENTS > requirements.txt
```
Comment (skyline75489) [inline .circleci/scripts/build_for_windows.sh:35]: This is ugly. But CircleCI does not seem to allow the one-liner I did at first.

**Comment by skyline75489** [comment]
@malfet @mrshenli I think this is ready. I don't fully understand why `pySoundFile` is needed on Windows. But my initial goal is completed. The requirements are converged into one `requirements.txt.

**Comment by mrshenli** [comment]
Hey @brianjo, shall we land this PR?

**Comment by brianjo** [comment]
> Hey @brianjo, shall we land this PR?

Sure. I'll do it right after we pass the rebase tests. Thanks!

**Comment by skyline75489** [comment]
The build failure seems unrelated to this PR. @brianjo is there a way for me to trigger the build manually?

**Comment by brianjo** [comment]
I’ll kick it again later on. Thanks!


________________________________

**Pull request merged**